joomla/joomla-cms
# [4.0] Messages/Alerts: using icons instead of text as heading

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30404

### Summary of Changes
As title says, replacing text by icons in Alerts


### Testing Instructions
Add to Atum and Cassiopea index.php

```
Factory::getApplication()->enqueueMessage('error message','error');
Factory::getApplication()->enqueueMessage('another long error message','error');
Factory::getApplication()->enqueueMessage('a danger message','danger');
Factory::getApplication()->enqueueMessage('warning message','warning');;
Factory::getApplication()->enqueueMessage('notice message','notice');
Factory::getApplication()->enqueueMessage('info message','info');
Factory::getApplication()->enqueueMessage('success message','success');
Factory::getApplication()->enqueueMessage('message no type');
```
in order to test all kinds of messages

Patch, Run NPM CI

### Actual result BEFORE applying this Pull Request

<img width="1239" alt="Screen Shot 2020-08-30 at 09 38 28" src="https://user-images.githubusercontent.com/869724/91653880-a9a63680-eaa4-11ea-9be7-bc630cbb8032.png">


### Expected result AFTER applying this Pull Request
Cassiopea

<img width="883" alt="Screen Shot 2020-08-30 at 09 23 41" src="https://user-images.githubusercontent.com/869724/91653807-2684e080-eaa4-11ea-835b-fa80e3319fa7.png">

Atum

<img width="1238" alt="Screen Shot 2020-08-30 at 09 23 22" src="https://user-images.githubusercontent.com/869724/91653809-28e73a80-eaa4-11ea-834f-12e2d87c560a.png">



### Documentation Changes Required
